client/src/components/FooterLinks/FooterLinks.tsx

```diff
@@ -2,6 +2,9 @@ import { Text, Container, Image, ActionIcon, Group, rem } from '@mantine/core';
 import { IconBrandInstagram, IconBrandFacebook } from '@tabler/icons-react';
 import logoImage from '../../images/logo.png';
 import classes from './FooterLinks.module.css';
+import React, { useState, ChangeEvent, FormEvent } from 'react';
```

Comment:
Remove `React`
_src/components/FooterLinks/FooterLinks.tsx:5:8 - error TS6133: 'React' is declared but its value is never read._